ui/page/exchange/create_order_page.go

```diff
@@ -1448,3 +1459,29 @@ func (pg *CreateOrderPage) fetchInstantExchangeCurrencies() error {
 	pg.rateError = err != nil
 	return err
 }
+
+// AssetToCreate check if there is any asset type that has not been created
+// and returns the first one.
+func (pg *CreateOrderPage) AssetToCreate() libutils.AssetType {
+	assetToCreate := pg.AssetsManager.AllAssetTypes()
+	wallets := pg.AssetsManager.AllWallets()
+
+	assetsNotCreated := make([]libutils.AssetType, 0)
+
+	for _, asset := range assetToCreate {
+		assetExists := false
+
+		for _, wallet := range wallets {
+			if wallet.GetAssetType() == asset {
+				assetExists = true
+				break
+			}
+		}
+
+		if !assetExists {
+			assetsNotCreated = append(assetsNotCreated, asset)
+		}
+	}
+
+	return assetsNotCreated[0]
```

Comment:
I think we should check the `len`? better safe than sorry.

Reply:
if len is not greater than zero then we would gave to return nil, which still panics

before this method can be called initially it means the app only has one asset created only, so this would always return a value

Reply:
ok